maps/SoulHouse.asm

```diff
@@ -14,7 +14,7 @@ SoulHouse_MapScriptHeader:
 
 	def_object_events
 	object_event  9,  7, SPRITE_AGATHA, SPRITEMOVEDATA_STANDING_UP, 0, 0, -1, -1, 0, OBJECTTYPE_SCRIPT, 0, AgathaScript, -1
-	object_event  4,  2, SPRITE_GRAMPS, SPRITEMOVEDATA_STANDING_DOWN, 0, 0, -1, -1, PAL_NPC_BLUE, OBJECTTYPE_COMMAND, jumptextfaceplayer, MrFujiText, -1 ; TODO: EVENT_SOUL_HOUSE_MR_FUJI
+	object_event  4,  2, SPRITE_ELDER, SPRITEMOVEDATA_STANDING_DOWN, 0, 0, -1, -1, PAL_NPC_BLUE, OBJECTTYPE_COMMAND, jumptextfaceplayer, MrFujiText, -1 ; TODO: EVENT_SOUL_HOUSE_MR_FUJI
```

Comment:
Mr. Fuji does not, in any canon source material, look like an Elder. He actually looks closer to the generic old guy.

Reply:
The Dragon Clan elder reuses his sprite from Gen 1, though.

Reply:
You're absolutely right. I was thinking of the trainer pic which has a beard. The overworld sprite doesn't. This is actually a very good suggestion. Sorry!

Reply:
That makes sense, thanks.
![image](https://user-images.githubusercontent.com/35663410/233869629-1983a80c-385c-43c2-a1fc-d2b7b825096e.png)

Reply:
There aren't any other maps Mr. Fuji appears in, are there?